app/presenters/step_by_step_page_presenter.rb

```diff
@@ -57,6 +57,20 @@ def summary_list_params
     params
   end
 
+  def broken_link_status(step)
+    return step.title unless step.link_report?
+
+    "
+      #{step.title}
+      <div class=\"step-broken-links__summary\">
```

Comment:
Agreed, this wasn't the way I wanted to approach it.

The problem is, the design has SVG icons and red/green status text:

![Screen Shot 2019-09-09 at 15 49 56](https://user-images.githubusercontent.com/5111927/64541268-7e698b00-d319-11e9-82aa-b35e0d76c392.png)

...so we can no longer just pass the broken link summary like this:

```ruby
step_config = {
  field: "1",
  value: "1 broken link"
}
```

The component would need to take something like this:

```ruby
step_config = {
  field: step.logic == 'number' ? step_number : step.logic,
  value: "
    <div><svg>...</svg> <span class='error'>1 broken link</span></div>
  ".html_safe
}
```

Ideally this would live in _broken-links-summary.html something like:

```ruby
step_config = {
  field: step.logic == 'number' ? step_number : step.logic,
  value: render 'shared/steps/broken-links-summary', step: step
}
```

But I haven't figured out how to make `render` accessible from the presenter 🤔 any ideas?

Reply:
I don't understand why you need to call render from the presenter.

Reply:
We want the content of `broken-links-summary.html` as our `value`. This is what I was hoping to pair on.

Yes I think in hindsight the best approach is to put it all in the view 👍 just means all the logic for the `step` method will be in the view too.

Reply:
Have now pulled this back out of the presenter and into the view.